Shared/Samples/Create buffers around points/CreateBuffersAroundPointsView.swift

```diff
@@ -0,0 +1,306 @@
+        
+        /// A 'Bool' value indicating whether the input box is showing.
+        @Published var inputBoxIsPresented = false
+        
+        /// The input obtained from the user for the buffer radius of a point.
+        @Published var radiusInput: String = ""
+        
+        /// The status of the sample.
+        @Published var status = Status.addPoints
+        
+        /// A 'Map' centered on Texas with image layers.
+        let map: Map
+        
+        /// An 'Array' for the graphic overlays.
+        var graphicsOverlays: [GraphicsOverlay] = []
+        
+        /// An 'Array' for the tapped points and their radii.
+        var bufferPoints: [(point: Point, radius: Double)] = []
+        
+        /// A 'GraphicsOverlay' for the boundary around the valid area of use.
+        private var boundaryGraphicsOverlay: GraphicsOverlay
+        
+        /// A 'GraphicsOverlay' for the points' buffers.
+        private var bufferGraphicsOverlay: GraphicsOverlay
+        
+        /// A 'GraphicsOverlay' for the points of the tapped locations.
+        private var tapPointsGraphicsOverlay: GraphicsOverlay
+        
+        /// A 'Polygon' that represents the valid area of use for the spatial reference.
+        private let boundaryPolygon: ArcGIS.Polygon
+        
+        init() {
+            /// The 'SpatialReference' for the sample.
+            let statePlaneNorthCentralTexas = SpatialReference(wkid: WKID(32038)!)!
+            
+            // Create boundary polygon.
+            boundaryPolygon = {
+                let boundaryPoints = [
+                    Point(latitude: 31.720, longitude: -103.070),
+                    Point(latitude: 34.580, longitude: -103.070),
+                    Point(latitude: 34.580, longitude: -94.000),
+                    Point(latitude: 31.720, longitude: -94.000)
+                ]
+                let polygon = GeometryEngine.project(Polygon(points: boundaryPoints), into: statePlaneNorthCentralTexas)!
+                return polygon
+            }()
+            
+            // Create map.
+            map = CreateBuffersAroundPointsView.Model.makeMap(
+                spatialReference: statePlaneNorthCentralTexas, 
+                viewpointGeometry: boundaryPolygon
+            )
+            
+            // Create graphics overlays.
+            boundaryGraphicsOverlay = CreateBuffersAroundPointsView.Model.makeBoundaryGraphicsOverlay(
+                boundaryGeometry: boundaryPolygon
+            )
+            bufferGraphicsOverlay = CreateBuffersAroundPointsView.Model.makeBufferGraphicsOverlay()
+            tapPointsGraphicsOverlay = CreateBuffersAroundPointsView.Model.makeTappedPointsGraphicsOverlay()
+            graphicsOverlays.append(
+                contentsOf: [boundaryGraphicsOverlay, bufferGraphicsOverlay, tapPointsGraphicsOverlay]
+            )
+        }
+        
+        /// Create a 'Map' with image layers from a 'SpatialReference'.
```

Comment:
```suggestion
        /// Creates a map with image layers from a spatial reference.
```
Applies to other make methods as well.